jpegutils.h

```diff
@@ -11,50 +11,8 @@
 #ifndef __JPEGUTILS_H__
 #define __JPEGUTILS_H__
 
- /*
-  * jpeg_data:       buffer with input / output jpeg
-  * len:             Length of jpeg buffer
-  * itype:           Y4M_ILACE_NONE: Not interlaced
-  *                  Y4M_ILACE_TOP_FIRST: Interlaced, top-field-first
-  *                  Y4M_ILACE_BOTTOM_FIRST: Interlaced, bottom-field-first
-  * ctype            Chroma format for decompression.
-  *                  Currently always 420 and hence ignored.
-  * raw0             buffer with input / output raw Y channel
-  * raw1             buffer with input / output raw U/Cb channel
-  * raw2             buffer with input / output raw V/Cr channel
-  * width            width of Y channel (width of U/V is width/2)
-  * height           height of Y channel (height of U/V is height/2)
-  */
+int jpgutl_decode_jpeg (unsigned char *jpeg_data, int len,
+                     unsigned int width_in, unsigned int height_in, unsigned char *img);
```

Comment:
I'll add some and revise the variable names.  

And now I'll digress.... :)

I personally find that many of those descriptions are extremely redundant and just clutter up the code.  e.g.  Describing width / height as what was included in the original or having to describe the "jpeg_data" parameter passed to the function jpgutl_decode_jpeg as "The jpeg data to decode" is just redundant and there is very little additional information that I can think of to add to the description.  

Now if the function is called "process" or the variables are called "img" or "var1" or "len" that is different and mandates a description or a better name.    i.e. Variables and functions should be named such that they _generally_ do not need an associated translation as to their contents or purpose.  Documentation should address the Why and How questions that are not immediately apparent.

Finally, adding the documentation back into the header file rather than the code base seem really questionable.  I certainly didn't go looking in the header for documentation when debugging this code.